open_ce/conda_utils.py

```diff
@@ -72,3 +75,27 @@ def get_output_file_paths(meta, variants):
         result.append(os.path.join(path.parent.name, path.name))
 
     return result
+
+def conda_package_info(channels, package):
+    '''
+    Get conda package info.
+    '''
+    pkg_args = "\"{}\"".format(generalize_version(package))
+    channel_args = " ".join({"-c \"{}\"".format(channel) for channel in channels})
+
+    cli = "conda search --json {} {} --info".format(channel_args, pkg_args)
+    ret_code, std_out, _ = run_command_capture(cli, stderr=subprocess.STDOUT)
+    if not ret_code:
+        raise OpenCEError(Error.CONDA_PACKAGE_INFO, cli, std_out)
+    return std_out
+
+def get_latest_package_info(channels, package):
+    '''
+    Get the conda package info for the most recent search result.
+    '''
+    results = yaml.safe_load(conda_package_info(channels, package))
+    retval = results[list(results.keys())[0]][0]
```

Comment:
Can probably store the result of `results[list(results.keys())[0]]` since it is used twice and kind of hard to read.

Reply:
That makes sense.